packages/wp-story-editor/src/api/taxonomy.js

```diff
@@ -23,21 +23,46 @@ import { addQueryArgs } from '@web-stories-wp/design-system';
  * WordPress dependencies
  */
 import apiFetch from '@wordpress/api-fetch';
+/**
+ * Internal dependencies
+ */
+import { snakeCaseToCamelCase } from './utils';
 
 /**
  * Get all taxonomies.
  *
  * @param {string} postType Post type.
  * @param {string} path API path.
+ * @param {Object} options Options to transform the data.
+ * @param {Object} options.asCamelCase Response properties will be in camel case.
  * @return {Promise} Taxonomies promise.
  */
-export function getTaxonomies(postType, path) {
-  return apiFetch({
+export async function getTaxonomies(
+  postType,
+  path,
+  { asCamelCase = false } = {}
```

Comment:
What's the reason for adding this param? 🤔 

I'd remove this param and just always return in camel case, really.